share/styles/Edwin.mss

```diff
@@ -62,6 +62,8 @@
     <staffFontFace>Edwin</staffFontFace>
     <rehearsalMarkFontFace>Edwin</rehearsalMarkFontFace>
     <rehearsalMarkFontSize>14</rehearsalMarkFontSize>
+    <rehearsalMarkFrameWidth>0.16</rehearsalMarkFrameWidth>
+    <rehearsalMarkFrameRound>0</rehearsalMarkFrameRound>

```

Comment:
Why? it is the new default anyhow, wouldn't it need to go to one of the pre3.6 mss files instead?

Reply:
That seems to cause at least one of the mtest failures

Reply:
Edwin.mss is the subset of the new defaults that's applied if you tick the 'use Edwin' box in the score migration dialog; so it contains all settings related to updated text styles. (Or am I misunderstanding you?)

Reply:
OK. So it is not the one takes when "Reset all setting to default".
Still it might be good to add those former settings to the pre 3.6 mss files?

Reply:
It looks to me like they are already in there (legacy-style-defaults-v1.mss, legacy-style-defaults-v2.mss and legacy-style-defaults-v3.mss), with the former values (0.2 and 20).

Reply:
OK, then I rest my case, thanks for checking